front/src/Phaser/Game/GameScene.ts

```diff
@@ -167,7 +167,7 @@ export class GameScene extends DirtyScene {
         value: RoomJoinedMessageInterface | PromiseLike<RoomJoinedMessageInterface>
     ) => void;
     // A promise that will resolve when the "create" method is called (signaling loading is ended)
-    private createPromise: Promise<void>;
+    public createPromise: Promise<void>;
```

Comment:
```suggestion
    private createPromise: Promise<void>;
```

We can keep this private.